src/components/CartItem/index.js

```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image';
+
+import useStore from '../../hooks/useStore';
+import StyledButton from '../Button/styled';
+import StyledProductName from '../ProductName/styled';
+import StyledProductPrice from '../ProductPrice/styled';
+
+import StyledCartItem from './styled';
+
+export default function CartItem({productid, id, imgUrl, name, price}) {
+	const deleteFromCart = useStore(state => state.deleteFromCart);
+
+	return (
+		<StyledCartItem key={id}>
+			<Image
+				alt={'Photo from the product'}
```

Comment:
I would recommend to leave the `alt` text empty as right now every product would have the same `alt` text. This would not be accessible nor good for SEO. Since it is only a decorative image, I would leave the attribute empty.
```suggestion
				alt=""
```